File: falleaves-server/falleaves-starter-model/src/main/java/org/techas/falleaves/model/UserLoginEntity.java
package org.techas.falleaves.model;

import javax.persistence.*;

@Entity
@Table(name = "user_login", schema = "falleaves", catalog = "")
public class UserLoginEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long uid;
    private String identifier;
    private String credential;

    public Long getId() {
        return id;
    }

    public UserLoginEntity setId(Long id) {
        this.id = id;
        return this;
    }

    public Long getUid() {
        return uid;
    }

    public UserLoginEntity setUid(Long uid) {
        this.uid = uid;
        return this;
    }

    public String getIdentifier() {
        return identifier;
    }

    public UserLoginEntity setIdentifier(String identifier) {
        this.identifier = identifier;
        return this;
    }

    public String getCredential() {
        return credential;
    }

    public UserLoginEntity setCredential(String credential) {
        this.credential = credential;
        return this;
    }
}

File: falleaves-server/auth/src/main/java/cn/falleaves/auth/model/UserLoginEntity.kt
package cn.falleaves.auth.model

import javax.persistence.*

@Entity
@Table(name = "user_login", schema = "falleaves", catalog = "")
open class UserLoginEntity {

    @get:Id
    @get:Column(name = "id", nullable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    open var id: Int? = null

    @get:Basic
    @get:Column(name = "uid", nullable = true)
    open var uid: Int? = null

    @get:Basic
    @get:Column(name = "types", nullable = false)
    open var types: Byte? = null

    @get:Basic
    @get:Column(name = "identifier", nullable = false)
    open var identifier: String? = null

    @get:Basic
    @get:Column(name = "credential", nullable = false)
    open var credential: String? = null

}

